modular_skyrat/master_files/code/game/objects/items/RCD.dm

```diff
@@ -8,3 +8,28 @@
 	matter = 500
 	canRturf = TRUE
 	upgrade = RCD_UPGRADE_FRAMES | RCD_UPGRADE_SIMPLE_CIRCUITS | RCD_UPGRADE_FURNISHING
+
+/obj/item/construction/plumbing/mining
+	name = "mining plumbing constructor"
+	desc = "A type of plumbing constructor designed to harvest from geysers and collect their fluids."
+	icon = 'modular_skyrat/modules/liquids/icons/obj/tools.dmi'
+	icon_state = "plumberer_mining"
```

Comment:
Is this misspelled?

Reply:
which?

Reply:
"plumberer"

Reply:
That's not a typo. That's how it exists in the .dmi file.